plugins/forc-index/src/cli.rs

```diff
@@ -32,6 +32,7 @@ pub enum ForcIndex {
     Build(BuildCommand),
     Auth(AuthCommand),
     Postgres(ForcPostgresOpt),
+    Welcome(WelcomeCommand),
```

Comment:
@rvmelkonian (per my most recent comment responding to @deekerno with regard to hiding the command from `--help` I'm merely suggesting we just comment this (line 35), and line 54 - and possibly add an `#[allow(unused)]` above so that clippy doesn't complain about the import

Reply:
Ah yes, I missed this comment. Will do.